components/StyledDiv.js

```diff
@@ -15,6 +15,11 @@ const StyledDiv = styled.div`
       padding: 1rem 1rem 1rem 1rem;
```

Comment:
If you want the same padding for all four edges you can write 1rem once and it will apply to all 🙂